src/azure-cli/azure/cli/command_modules/eventgrid/commands.py

```diff
@@ -59,10 +122,60 @@ def load_command_table(self, _):
         g.custom_command('list', 'cli_domain_list')
         g.custom_command('create', 'cli_domain_create_or_update')
         g.command('delete', 'delete')
-        g.generic_update_command('update',
-                                 getter_name='get',
-                                 setter_name='update',
-                                 client_factory=domains_factory)
+        g.custom_command('update', 'cli_domain_update')
+
+    with self.command_group('eventgrid system-topic', system_topics_mgmt_util, client_factory=system_topics_factory, is_preview=True) as g:
+        g.show_command('show', 'get')
+        g.command('delete', 'delete')
```

Comment:
could we add confirmation=True for all delete commands?

Reply:
No. We keep consistency with all other existing delete commands.